docs/optimization/summary.md

```diff
@@ -0,0 +1,13 @@
+# Performance
+
+Once you get your desired circuit, you might want to analyze your circuit and optimize it in terms of execution time. Here are some quick tips to do that:
+
+- Reduce the amount of [table lookups](../core-features/table_lookups.md) in the circuit.
+- Try different implementation strategies for [complex operations](../core-features/non_linear_operations.md#comparisons).
+- Utilize [rounding](../core-features/rounding.md) and [truncating](../core-features/truncating.md) if your application doesn't require precise execution.
+- Use tensors as much as possible in your circuits.
+- Enable dataflow parallelization.
```

Comment:
I would go for:

"Enable dataflow parallelization, by setting [this option] = True"

Reply:
Done.